suitcase/tiff_series/__init__.py

```diff
@@ -164,11 +168,42 @@ class Serializer(tiff_stack.Serializer):
     """
     def __init__(self, directory, file_prefix='{start[uid]}-', bigtiff=False,
                  byteorder=None, imagej=False, **kwargs):
+
         super().__init__(directory, file_prefix, bigtiff,
                          byteorder, imagej, **kwargs)
         # maps stream name to dict that map field name to index (#)
         self._counter = defaultdict(lambda: defaultdict(itertools.count))
 
+    def start(self, doc):
+        '''Extracts `start` document information for formatting file_prefix.
+
+        This method checks that only one `start` document is seen and formats
+        `file_prefix` based on the contents of the `start` document.
+
+        Parameters:
+        -----------
+        doc : dict
+            RunStart document
+        '''
+
+        # raise an error if this is the second `start` document seen.
+        if self._start:
+            raise RuntimeError(
+                "The serializer in suitcase.tiff expects documents from one "
+                "run only. Two `start` documents where sent to it")
+        else:
+            self._start = doc  # record the start doc for later use
+            if isinstance(self._directory, (str, Path)):
+                try:  # if self._directory is a string
+                    self._templated_directory = self._directory.format(
```

Comment:
Do we have the `_templated_directory` attribute initialized anywhere?

Reply:
yes, as this is a child of tiff_stack.serializer then it is defined there

Reply:
Got it, sounds good.